perception/image_projection_based_fusion/config/pointpainting.param.yaml

```diff
@@ -26,7 +26,7 @@
       iou_nms_target_class_names: ["CAR"]
       iou_nms_search_distance_2d: 10.0
       iou_nms_threshold: 0.1
-      score_threshold: 0.4
+      score_threshold: 0.35
```

Comment:
(Not related to this part) Would you mind modifying awf/autoware_launch as well?
https://github.com/autowarefoundation/autoware_launch/blob/b23c5f948421037d2bf81dd340015c4b2fd1602c/autoware_launch/config/perception/object_recognition/detection/lidar_model/pointpainting.param.yaml#L29

Reply:
OK, I will make the PR to awf/autoware_launch!